hack/validation/test_release.py

```diff
@@ -29,7 +29,7 @@ def test_cslb_date(self):
                           msg=f"Row ID {row['entity:cslb_id']} does not have a 2020 cslb_date")
 
     def test_spay_or_neuter_age(self):
-        for row in self._hles_dog2_data:
+        for row in self._hles_dog1_data:
```

Comment:
🙈 out of scope but: we need to get rid of this switcheroo-ing, I'll poke at that this week.